src/set_theory/game/pgame.lean

```diff
@@ -765,47 +762,61 @@ end
 lemma zero_add_equiv (x : pgame.{u}) : 0 + x ≈ x :=
 (zero_add_relabelling x).equiv
 
-/-- An explicit equivalence between the moves for Left in `x + y` and the type-theory sum
-    of the moves for Left in `x` and in `y`. -/
-def left_moves_add (x y : pgame) : (x + y).left_moves ≃ x.left_moves ⊕ y.left_moves :=
-by { cases x, cases y, refl, }
+@[simp] theorem left_moves_add (x y : pgame.{u}) :
+  (x + y).left_moves = (x.left_moves ⊕ y.left_moves) :=
+by { cases x, cases y, refl }
```

Comment:
As with the PR, this probably isn't useful as `@[simp]`, and I'm not entirely sold that it is better as an equality.

Reply:
I made it no longer `simp`. I'd still like to keep it as a lemma, can't see that doing any harm (besides, makes the equivalences' definitions more direct).